Gruntfile.js

```diff
@@ -74,5 +88,12 @@ module.exports = function(grunt) {
   }
 
   grunt.registerTask("dev", ["connect", "watch"]);
-  grunt.registerTask("test", ["connect", "saucelabs-custom"]);
+  grunt.registerTask("test-locally", ["connect", "jasmine"]);
```

Comment:
It's a bit odd that you have a separate task for testing locally, but not for saucelabs as well. I would restructure the tasks as follows:

```js
grunt.registerTask("test-phantomjs", ["connect", "jasmine"]);
grunt.registerTask("test-saucelabs", ["connect", "saucelabs-custom"]);
```

And then just have the `if` below register `"test"` for either `"test-phantomjs"` or `"test-saucelabs"`